torch/csrc/jit/passes/normalize_ops.cpp

```diff
@@ -20,6 +20,19 @@ bool normalizeOpAliases(graph_node_list_iterator& iter) {
   return false;
 }
 
+// Normalize rsub such that `rsub(x,y) = sub(x,y)`
+bool normalizeRSub(graph_node_list_iterator& iter) {
+  ArrayRef<Value*> args = iter->inputs();
+  if (iter->kind() == aten::rsub) {
+    iter->replaceWithNewSymbol(aten::sub);
```

Comment:
`replaceWithNewSymbol` returns the updated node, so the replacements we're doing in the following lines will not affect it but the existing node we're destroying